### ISSUE TITLE
Problems with samtools versions (1.10)

### ISSUE THREAD

**Issue by a-lapin** [tseemann/snippy#344]
Hi, I've recently upgraded samtools to version 1.10 via linuxbrew. Since this update snippy is broken, probably it presumes that v.1.10 < v.1.7
```

[11:40:06] Found vt - /home/linuxbrew/.linuxbrew/bin/vt
[11:40:06] Found snippy-vcf_to_tab - /home/linuxbrew/.linuxbrew/Cellar/snippy/4.4.5/bin/snippy-vcf_to_tab
[11:40:06] Found snippy-vcf_report - /home/linuxbrew/.linuxbrew/Cellar/snippy/4.4.5/bin/snippy-vcf_report
[11:40:06] Need samtools --version >= 1.7 but you have 1.10 - please upgrade it.
```

**Comment by tseemann** [comment]
Yes, it is a floating point comparison bug.

Please run`brew switch samtools 1.9` until I can fix it.

**Comment by andersgs** [comment]
Remember to run `brew switch htslib 1.9` and `brew switch bcftools 1.9_1` too.

**Comment by chienchi** [comment]
I tried to fixed it by switching to a dotted-decimal $VERSION for comparison using "version" module.  But not sure it is the correct way. 
```
use version;

97 err("Need $cmd >= $min but you have $ver - please upgrade it.") if version->declare($ver) < version->declare($min);
```

**Comment by tseemann** [comment]
@chienchi that is a great core perl solution! i was thinking i would need `SymVer.pm` but your method is better.

**Comment by luciagrami** [comment]
Hello, I have the same issue but 
$ samtools --version
 throws:

samtools 1.9
Using htslib 1.9
Copyright (C) 2018 Genome Research Ltd.

And $brew switch samtools 1.9 gives an error:

Error: samtools not found in the Cellar.

**Comment by Abgarmar** [comment]
Same issue too after installing snippy via conda:

conda install -c conda-forge -c bioconda -c defaults snippy

Need samtools --version >= 1.7 but you have 1.10 - please upgrade it

samtools 1.10
Using htslib 1.10.1
Copyright (C) 2019 Genome Research Ltd.

**Comment by quocviet0908** [comment]
Hi everyone,

I just got stuck in this problem as well and I think this solution can help everyone.
After installing snippy v4.4.5, this version should come along with samtools and bcftools 1.10. You can downgrade samtools to 1.9 easily by running "conda install -c bioconda samtools=1.9" inside the snippy environment.

Cheer!

**Comment by a-lapin** [comment]
Hi!
I encountered a similar error with prokka and makeblastdb:

```
(base) argive@argive-PC:~/Desktop$ prokka --setupdb
[13:37:41] Appending to PATH: /home/linuxbrew/.linuxbrew/Cellar/prokka/1.14.0/bin
[13:37:41] Cleaning databases in /home/linuxbrew/.linuxbrew/Cellar/prokka/1.14.0/db
[13:37:41] Cleaning complete.
[13:37:41] Looking for 'makeblastdb' - found /home/linuxbrew/.linuxbrew/bin/makeblastdb
[13:37:41] Determined makeblastdb version is 2.10
[13:37:41] Prokka needs makeblastdb 2.8 or higher. Please upgrade and try again.
```

**Comment by tseemann** [comment]
This bug will be fixed in snippy 4.5.0
